sample/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,6 @@
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#00574B</color>
     <color name="colorAccent">#D81B60</color>
+    <color name="black">#000000</color>
+    <color name="white">#ffffff</color>
```

Comment:
```suggestion
    <color name="color_showcase_black">#000000</color>
    <color name="color_showcase_white">#ffffff</color>
```

Reply:
These resources are not used anywhere. Why you suggest renaming?

Reply:
Then delete.